modules/custom/az_media/config/install/core.entity_view_mode.media.az_thumbnail_100x100.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media
+id: media.az_thumbnail_100x100
```

Comment:
Is this view mode a duplicate of https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_media/config/install/core.entity_view_mode.media.az_thumbnail.yml ?

Reply:
Nope lol

Reply:
@bberndt-uaz To look into this

Reply:
@danahertzberg I think you can just set the media view display for the existing `Thumbnail (100 x 100)` view mode instead of the new `AZ Thumbnail (100x100)` view mode at `/admin/structure/media/manage/az_image/display`.

<img width="510" alt="image" src="https://github.com/user-attachments/assets/ea6b6917-e0f3-43bf-b329-0f691a9726f8">
<p> <p>

You can probably just edit `/modules/custom/az_media/config/install/core.entity_view_display.media.az_image.az_thumbnail_100x100.yml` to remove the `_100x100` from the filename and YAML (lines 5 and 124 I think).